collect_app/src/main/res/layout/ready_to_send_banner.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/banner"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?colorSurfaceContainerLow"
+    android:paddingVertical="@dimen/margin_standard"
+    android:visibility="gone"
+    tools:visibility="visible">
+
+    <ImageView
+        android:id="@+id/icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_large"
+        app:srcCompat="@drawable/ic_send_24"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:tint="?colorPrimary"/>
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_large"
+        android:layout_marginEnd="@dimen/margin_standard"
+        android:textAppearance="@style/TextAppearance.Material3.TitleMedium"
```

Comment:
This should use the theme attribute (`?textAppearanceTitleMedium`) rather than the style directly so that it can be overridden in the theme.

Reply:
If I use that the text is very small:
![Screenshot_1693300262](https://github.com/getodk/collect/assets/3276264/a881d62a-7d45-4f07-ac9a-35dba22d09cc)
Maybe it will be fixed once https://github.com/getodk/collect/pull/5676 is merged?

Reply:
We have merged https://github.com/getodk/collect/pull/5676 and after rebasing it works well.